src/main/scala/io/iohk/ethereum/network/p2p/messages/CommonMessages.scala

```diff
@@ -111,10 +118,23 @@ object CommonMessages {
     )
 
     val FirstByteOfAddress = 12
-    val LastByteOfAddress = 32
+    val LastByteOfAddress = FirstByteOfAddress + AddressLength
```

Comment:
this constants are here to get `recoveredAddress` which is part of transaction, but maybe they should be private or moved somewhere?